var/spack/repos/builtin/packages/py-pathos/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPathos(PythonPackage):
+    """Parallel graph management and execution in heterogeneous computing """
+
+    homepage = "https://github.com/uqfoundation/pathos"
+    url      = "https://pypi.io/packages/source/p/pathos/pathos-0.2.0.zip"
+
+    version('0.2.0', '7a840ce6c3a67d71e6ad7339034ec53e')
+
+    depends_on('py-setuptools@0.6:', type='build')
+    depends_on('py-multiprocess@0.70.5:', type=('build', 'run'))
```

Comment:
@adamjstewart What are the guidelines that you use to determine the versions?

Most part of versions I put come from the [setup.py](https://github.com/uqfoundation/pathos/blob/master/setup.py#L211), except for some that escaped to me, as in `py-multiprocess`